django_loci/channels/base.py

```diff
@@ -3,6 +3,7 @@
 from django.core.exceptions import ValidationError
 
 location_broadcast_path = "ws/loci/location/<uuid:pk>/"
+all_location_broadcast_path = "ws/loci/location/all/"
```

Comment:
Instead of `all`, can we use `common_location_broadcast_path`?

We can also follow the REST naming convention, but we will need to do that in backward compatible fashion. We can create a separate issue for this.